airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/63cea06f-1c75-458d-88fe-ad48c7cb27fd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sourceDefinitionId": "63cea06f-1c75-458d-88fe-ad48c7cb27fd",
+  "name": "Braintree",
```

Comment:
Could you add "Deprecated" to the old one (for `json` and `yml` files)

Reply:
Done